### ISSUE TITLE
vscode freezes when extended to multiple screen in Win11

### ISSUE THREAD

**Issue by andylokandy** [microsoft/vscode#209249]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.87.2 (user setup)
Commit: 863d2581ecda6849923a2118d93a088b0745d9d6
Date: 2024-03-08T15:20:17.278Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.22631

Steps to Reproduce:

1. Having two monitors side by side.
2. Extend the window across two monitors, where the left one is the main (1st) monitor.
3. For every 5s, the vscode in the right monitor will not get refreshed, which means when I scroll down in the left screen, the part in the right monitor will be stuck there.

Temporary fixes:
1. Move the window around, then the right screen will immediately get refreshed.
2. Or set the right monitor as the main monitor.

Related: https://github.com/microsoft/vscode/issues/197451

**Comment by joao-aguirre** [comment]
Same problem here, the temporary solution dramatically reduced the number of freezes but there are still some small hiccups.
I hope it is actually a temporary solution, as it messes up my setup.

**Comment by andylokandy** [comment]
@joao-aguirre I've figured out a third solution that won't mess up your setup: try to swap the monitor cables connecting to the pc, and then swap the virtual monitor placement order in system setting, and finally set the main monitor to the monitor on the side you're used to.

**Comment by joao-aguirre** [comment]
Hello @andylokandy, thank you for the suggestion!

Unfortunately it seems that I jumped the gun when saying that changing the main window to the other monitor solved the issue. Actually, it had just transferred the problem to that side of the VSCode window (where I usually leave the Output and debug panels).

I have tried changing the cables and moving the main monitor back to the left as you have suggested, it kept the freezing problem occurring on the right monitor, which is a bit better than having these freezes on the left one (where I keep the code editors), but they are still there anyway.
